src/Symfony/Component/Debug/Tests/Fixtures/ReturnTypeParent.php

```diff
@@ -0,0 +1,193 @@
+<?php
+
+namespace Symfony\Component\Debug\Tests\Fixtures;
+
+abstract class ReturnTypeParent extends ReturnTypeGrandParent implements ReturnTypeInterface
+{
+    // REAL RETURN TYPE CASES
+
+    // expected => NO DEPRECATION TRIGGERED
+
+    /**
+     * @return string
+     */
+    abstract public function realReturnTypeMustBeThere(): string;
+
+    /**
+     * @return float
+     */
+    public function realReturnTypeIsAlreadyThere()
+    {
+        // expected => NO DEPRECATION TRIGGERED
+    }
+
+    // expected => NO DEPRECATION TRIGGERED
+
+    /**
+     * @return iterable|null
+     */
+    abstract public function realReturnTypeIsAlreadyThereWithNull();
+
+    //DOC RETURN TYPE CASES
+
+    /**
+     * @return false
+     */
+    public function oneCommonNonObjectReturnedType()
+    {
+        // expected => NO DEPRECATION TRIGGERED
```

Comment:
Wouldn‘t we turn `@return false` into `: bool`?

Reply:
Yes! https://github.com/symfony/symfony/pull/30323#discussion_r311013017